keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -827,6 +827,18 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+- Start a local Kubernetes 1.26 cluster (`PodSchedulingReadiness` defaulted to false)
+- Create a Pod `pause` with one scheduling gate
+- The Pod's scheduling gate gets dropped as expected due to disabled feature gate
+- Delete the Pod
+- Re-start API Server and scheduler with version 1.27, and specify `PodSchedulingReadiness=true`
+- Create the same Pod `pause` with one scheduling gate
+- The Pod stays in `SchedulingGated` state, and its `.spec.schedulingGate` is persisted
+- Re-start API Server and scheduler with version 1.26
+- The Pod `pause` enters `Pending` state, with old `.spec.schedulingGate` reserved
```

Comment:
This is due to the change of https://github.com/kubernetes/kubernetes/pull/115569 - Jordan pointed out that we should reject setting nodeName to a pod that is scheduling gated. Otherwise, this pod would be in Running state.

Reply:
Yes - that sounds desired.